app/javascript/listings/components/Modal.jsx

```diff
@@ -16,38 +17,61 @@ const Modal = ({
 }) => {
   const shouldRenderMessageModal = listing && listing.contact_via_connect;
 
+  const Icon = () => (
+    <svg
+      width="24"
+      height="24"
+      viewBox="0 0 24 24"
+      className="crayons-icon pointer-events-none"
+      xmlns="http://www.w3.org/2000/svg"
+    >
+      <path d="M12 10.586l4.95-4.95 1.414 1.414-4.95 4.95 4.95 4.95-1.414 1.414-4.95-4.95-4.95 4.95-1.414-1.414 4.95-4.95-4.95-4.95L7.05 5.636l4.95 4.95z" />
+    </svg>
+  );
+
+  // TODO: Why are we not using the crayons modal component and instead recreating it here?
```

Comment:
@ludwiczakpawel, I left this as a TODO, but any reason we are not using the crayons Preact `<Modal />` component?

Reply:
TBH I'm learning to make my PRs smaller (:D), therefor I didn't want to add this <Modal /> dependency since I was a little scared it would cause some issues that I wouldn't be able to quickly solve with sensible amount of code... But I'm absolutely in favor of using it.

Reply:
https://github.com/forem/forem/pull/9349#discussion_r462552969